src/pages/settings/Profile/CustomStatus/CustomClearAfterPage.js

```diff
@@ -59,7 +59,7 @@ function CustomClearAfterPage({translate, customStatus}) {
                 shouldUseDefaultValue
             >
                 <NewDatePicker
-                    inputID="dob"
+                    inputID="dataTime"
```

Comment:
Sorry one more time, why `dataTime` instead of `dateTime`? 😅

Reply:
Fair enough 😄 
typo